templates/cli_cmd.gotmpl

```diff
@@ -0,0 +1,315 @@
+// Code generated by go-openrpc. DO NOT EDIT.
+
+/*
+Copyright © 2019 NAME HERE <EMAIL ADDRESS>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+    "strconv"
+    "strings"
+    "errors"
+  "fmt"
+  "log"
+  "github.com/spf13/cobra"
```

Comment:
Perhaps not for this PR, but if we're going to use cobra for the generated cli we should use it ourselves.

Reply:
Yea. I'd be happy to go with it if you are (but maybe next PR). Only question left is to understand license implications, as bespoke above.